src/EventStore.LogV3/Raw.cs

```diff
@@ -50,11 +49,20 @@ public struct RecordHeader {
 
 		[StructLayout(LayoutKind.Explicit, Size = RecordHeader.RecordIdSize, Pack = 1)]
```

Comment:
is there a specific reason for setting `Size = RecordHeader.RecordIdSize` instead of adding a constant `Size = 16` in the `StreamWriteId` struct? (they do not seem related)

Reply:
it does actually have to be the same size, because it occupies the RecordId section of the RecordHeader instead of the guid

Reply:
ok thanks :+1: